ext/standard/tests/assert/assert_basic6.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+assert() - Define null as assert callback
+--INI--
+assert.active=1
+--FILE--
+<?php
+
+function f1()
+{
+    echo "foo\n";
+}
+
+assert_options(ASSERT_CALLBACK);
+try {
+    assert(false);
+} catch (AssertionError $exception) {
+    echo $exception->getMessage() . "\n";
+}
+
+assert_options(ASSERT_CALLBACK, null);
+try {
+    assert(false);
+} catch (AssertionError $exception) {
+    echo $exception->getMessage() . "\n";
+}
+
+?>
+--EXPECTF--
+assert(false)
+
+Warning: Invalid callback , no array or string given in %s on line %d
```

Comment:
Heh, not really the behavior I would expect. I think null should just unset the callback.

Reply:
Yes, I was also a bit surprised. That said, can I change this behaviour? It doesn't make much sense.

Reply:
Yes, I would suggest to change it.

Reply:
Fixed :)